eng/Packages.Data.props

```diff
@@ -155,7 +155,7 @@
     All should have PrivateAssets="All" set so they don't become package dependencies
   -->
   <ItemGroup>
-    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20220701.3" PrivateAssets="All" />
+    <PackageReference Update="Microsoft.Azure.AutoRest.CSharp" Version="3.0.0-beta.20220701.2" PrivateAssets="All" />
```

Comment:
This seems like a down-grade, unless versioning in Autorest differs.  This seems like something we shouldn't do...